projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts

```diff
@@ -306,6 +306,12 @@ export class FhiAngularHighchartsComponent implements OnChanges {
 
     if (!diagramTypeIsDisabled) {
       this.updateDiagram();
+    } else {
+      console.warn(
```

Comment:
Jeg tenker kansje vi også bør sende ut en ren error med stack trace. Det vil potensielt gjøre det lettere for oss å debugge om det blir rapportert. Kan også være beleilig om det er en power user som bruker komponenten.

Hva tenker du?